cmd/root_options.go

```diff
@@ -26,6 +26,7 @@ type RootOptions struct {
 	KernelRelease    string `validate:"required,ascii" name:"kernel release"`
 	Target           string `validate:"required,target" name:"target"`
 	KernelConfigData string `validate:"omitempty,base64" name:"kernel config data"` // fixme > tag "name" does not seem to work when used at struct level, but works when used at inner level
+	BuilderImage     string `name:"builder image"`
```

Comment:
Would be cool to have a validator for it.

Most importantly, I think the default should be set here.

```suggestion
	BuilderImage     string `default:"falcosecurity/driverkit-builder:latest" name:"builder image"`
```

Feel free to pick my suggestion.

Reply:
I believe I wanted to do that but, please correct me if I'm wrong, when a new version of driverkit is released that string will be replaced with something that is not `falcosecurity/driverkit-builder:latest` but an image that has been generated at compile time for that specific version. I recall that by looking at the commit in the Makefile and in `buildprocessor.go`.
This is why the default we want is printed by looking at a variable in the appropriate module.

Reply:
https://github.com/falcosecurity/driverkit/blob/3ea18c6c767ac0c80102e469708102bd5bbc5e4f/cmd/root_options.go#L39 here with the `defaults` package setters.

Reply:
First of all, thanks a lot for validating the idea. This PR introduces code to do exactly what you wrote above in the yaml file and so I now know this is a good thing from the authors' perspective :)

Re. validator: makes sense, I can do it. 👍 
Re. "latest": when I read the driverkit source some time ago I tried to think about the reason why it was done like that, and I believe that it's designed to allow every version to keep working independently of changes in the moving "latest" tag. So if I have Driverkit v0.6.0 it will still work even if devs create a new builder for v0.7.0 which is incompatible with the previous one.
Re. injected tag: I think we want to retain this behavior. Do you have a better idea about how to do this cleanly? Thanks!

Reply:
Done as suggested (if I got it right)